mergify_engine/redis_utils.py

```diff
@@ -132,16 +137,21 @@ def queue(self) -> RedisQueue:
             config.QUEUE_URL,
             decode_responses=False,
             max_connections=self.queue_max_connections,
+            redis_connect_func=None,
         )
         return RedisQueue(client)
 
     @functools.cached_property
     def stream(self) -> RedisStream:
-        client = self.redis_from_url(
+        # Note(Syffe): mypy struggles to recognize the type of load_scripts because typeshed is missing
+        # typing on the objects we use here.
+        # cf: https://github.com/python/typeshed/pull/8147
+        client = self.redis_from_url(  # type: ignore[call-overload]
             "stream",
             config.STREAM_URL,
             decode_responses=False,
             max_connections=self.stream_max_connections,
+            redis_connect_func=load_scripts,
```

Comment:
```suggestion
            redis_connect_func=load_stream_scripts,
```
might be clearer so we know later we need to have a function per connection